sample/pytorch/utils/megatron_ckpt_convert.py

```diff
@@ -143,6 +143,16 @@ def split_and_convert_process(i, saved_dir,factor,key,args, val, ckpt_ver):
 
     elif key.find("attention.query_key_value.bias") != -1:
         local_dim = (int)(val.shape[-1] / 3)
+
+        if ckpt_ver == 3:
+            num_splits = 3
+            head_num = args.head_num
+            size_per_head = hidden_dim//head_num
```

Comment:
You use `hidden_dim` without defining the variable.

Reply:
Fixed it.